assets/base.css

```diff
@@ -154,17 +154,44 @@
 .color-icon-outline-button {
   --color-icon: rgb(var(--color-base-outline-button-labels));
 }
-
-.contains-card,
-.card {
-  --border-radius: var(--card-corner-radius);
-  --border-width: var(--card-border-width);
-  --border-opacity: var(--card-border-opacity);
-  --shadow-horizontal-offset: var(--card-shadow-horizontal-offset);
-  --shadow-vertical-offset: var(--card-shadow-vertical-offset);
-  --shadow-blur-radius: var(--card-shadow-blur-radius);
-  --shadow-opacity: var(--card-shadow-opacity);
-  --shadow-visible: var(--card-shadow-visible);
+ 
```

Comment:
For search results I see discussion about the page card types, but also wanted to make sure it's clear that this page could have a similar result to collage, where blog, product, and collection cards can all appear adjacent to each other in the same grid.
<img src="https://screenshot.click/22-56-yntl8-z5t7s.png" width="400"/>

Unless all the card types are configured the same, this will _usually_ create an ugly result. And unlike collage, I don't really have any ideas on how to feasibly avoid it. Are we ok with this?